### DISCUSSION TITLE
DBC-Wiki/dbc-cloud-cybercafe/diskless-netboot-server

### DISCUSSION THREAD

**Discussion by giscus** [DeepBrainChain/DBC-Wiki#57] [Announcements]
# DBC-Wiki/dbc-cloud-cybercafe/diskless-netboot-server

DBC-CloudCyberCafe 项目使用 iPXE 引导启动，使用 iSCSI 做无盘服务，使用 lvm2 做存储管理，使用 thrift 做 API 接口，使用 plotly 提供控制台界面，共同组合为客户提供无盘启动服务，因此部署无盘网起服务器也就是分别部署这些模块。服务器系统使用 ubuntu 20.04。 iPXE PXE 是 Intel...

https://deepbrainchain.github.io/DBC-Wiki/dbc-cloud-cybercafe/diskless-netboot-server.html

**Comment by tyab00** [comment]
您好！请问"无盘网起服务器"是免费使用的吗？还是哪些授权方面的限制？谢谢！
